File: Multi_Language_CodeExamples/dart/classes.dart
class Person {
  final String name;
  final int age;

  Person(this.name, this.age);

  void greet() {
    print("Hello, I'm $name and I'm $age years old");
  }
}

void main() {
  var person = Person("Eve", 26);
  person.greet();
}

File: Multi_Language_CodeExamples/python/classes.py
class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def greet(self):
        return f"Hello, I'm {self.name} and I'm {self.age} years old"

person = Person("Alice", 25)
print(person.greet())